grid/services/torch/hook_service.py

```diff
@@ -462,34 +453,33 @@ def hook_tensor(self, tensor_type):
         self.hook_get_(tensor_type)
         tu.hook_tensor__ser(self, tensor_type)
 
-
-    def hook_variable(self):
```

Comment:
Please return to the way it was before.  If we hook over torch.nn.Parameter or SharedVariable, we'll be double-hooking.

Reply:
Which way? I just accepted your changes. My apologies if I messed that up

Reply:
No worries, that may have been from an intermediate state.  `hook_variable` shouldn't accept a variable type argument, so any code that calls that function will need to be modified.